.github/workflows/hierarchy.yml

```diff
@@ -13,7 +13,7 @@ on:
 
 jobs:
   diff:
-    runs-on: [self-hosted]
+    runs-on: [ubuntu-latest]
```

Comment:
This change is also intended here?

Reply:
yep!

Reply:
Why? All the heavy stuff should run on our runners

Reply:
Ah It's just diff 🤦🏻‍♂️